frontend/src/screens/internal-apps/AlbyGo.tsx

```diff
@@ -0,0 +1,112 @@
+import { Globe } from "lucide-react";
+import { Link } from "react-router-dom";
+import AppHeader from "src/components/AppHeader";
+import ExternalLink from "src/components/ExternalLink";
+import { AppleIcon } from "src/components/icons/Apple";
+import { NostrWalletConnectIcon } from "src/components/icons/NostrWalletConnectIcon";
+import { PlayStoreIcon } from "src/components/icons/PlayStore";
+import { suggestedApps } from "src/components/SuggestedAppData";
+import { Button } from "src/components/ui/button";
+import {
+  Card,
+  CardContent,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from "src/components/ui/card";
+
+const ALBY_GO_APP_ID = "alby-go";
+
+export function AlbyGo() {
+  const app = suggestedApps.find((x) => x.id === ALBY_GO_APP_ID);
+
+  return (
+    <div className="grid gap-5">
+      <AppHeader
+        title={
+          <>
+            <div className="flex flex-row items-center">
+              <img src={app?.logo} className="w-14 h-14 rounded-lg mr-4" />
+              <div className="flex flex-col">
+                <div>{app?.title}</div>
+                <div className="text-sm font-normal text-muted-foreground">
+                  {app?.description}
+                </div>
+              </div>
+            </div>
+          </>
+        }
+        description=""
+        contentRight={
+          <Link to={`/apps/new?app=${app?.id}`}>
```

Comment:
Ideally we just show a qr code on the screen when you press this button rather than linking, so the user doesn't have to see all the wallet permissions options. But maybe we can do this as a second iteration